data/gc/monster_gc.lua

```diff
@@ -0,0 +1,15 @@
+function monster_gc_memory()
```

Comment:
djarek and I already taken care of that issue, it's clearing memory on /reload monsters just fine now.